2021/27xxx/CVE-2021-27551.json

```diff
@@ -1,18 +1,85 @@
 {
```

Comment:
CNA(s) of user (HCL) do not match owning CNA.

Reply:
Can you point to the mismatch?  Which fields?

Reply:
Sorry.  I do not understand the problem, so not sure how to resolve it.

Reply:
I see the error.  The CVE ID should be CVE-2022-27551.  Let me correct that